eng/targets/PackageDownloadAndReference.targets

```diff
@@ -0,0 +1,35 @@
+<Project>
+
+  <!--
+  Based on https://github.com/dotnet/runtime/blob/9893d145ebb61ae592afa1a64baa2f6f094b1f6b/eng/PackageDownloadAndReference.targets
+  
+  These file supports using PackageDownloadAndReference items.
+
+  The PackageDownloadAndReference item is used to download a package and reference it in the project, without restoring the package's dependency closure.
+
+  When using PackageDownloadAndReference you are responsible for selecting the correct assets from the package and ensuring that the package and it's
+  dependencies are available at runtime.
```

Comment:
Any chance a feature for this is on NuGet's backlog?

Reply:
There's https://github.com/NuGet/Home/issues/13821 as a very specific case (which also applies here), but we've also been discussing a more general story for plugins / excludeassets=runtime case.  In that case what you really want is reference assemblies for API provided by the host, then package-knockout for any package that overlaps.  It's on the list of things we want to solve for 10.0 - we think it's essential for making package updates work.